File: src/ua/lviv/iot/Chronicle.java
package ua.lviv.iot;

public class Chronicle extends Exhibit{
    private int  numberOfPages;
    
    public Chronicle() {
    	
    }
     public Chronicle(ExhibitName name, ExhibitAge age, String kindOfArt, String placeOfManufacture, String condition, double weight, int numberOfPages) {
    	     super(name, age, kindOfArt, placeOfManufacture, condition, weight);
    	     this.numberOfPages = numberOfPages;
     }
	public int getNumberOfPages() {
		return numberOfPages;
	}
	public void setNumberOfPages(int numberOfPages) {
		this.numberOfPages = numberOfPages;
	} 
	
	@Override
	public String toString() {
		return super.toString() + ", number of pages: " + this.numberOfPages + "\n";
	}
}

File: Lab_4/exhibits/Chronicle.py
from exhibits.Exhibit import *
from enums.ExhibitName import *
from enums.ExhibitAge import *


class Chronicle(Exhibit):
    exhibit_age = ExhibitAge.FIVE_BC
    exhibit_name = ExhibitName.CHRONICLE

    def __init__(self, number_of_pages, name, age, kind_of_art, place_of_manufacture, condition, weight):
        self.number_of_pages = number_of_pages
        Exhibit.__init__(self, name, age, kind_of_art, place_of_manufacture, condition, weight)

    def __str__(self):
        return "Name : " + str(self.name) + " age: " + str(self.age) + " the number of pages of chronicle is " + str(
            self.number_of_pages)